src/dex-ui/layouts/DashboardLayout.tsx

```diff
@@ -0,0 +1,178 @@
+    transactionState: location.state?.transactionState,
+  });
+
+  function onBackToDAOsLinkClick() {
+    navigate(Paths.DAOs.absolute);
+  }
+
+  if (isError) {
+    return <ErrorLayout message={errorMessage} />;
+  }
+
+  if (isLoading) {
+    return <LoadingSpinnerLayout />;
+  }
+
+  if (isNotFound) {
+    return (
+      <NotFound
+        message={`We didn't find any data for this DAO (${daoAccountId}).`}
+        preLinkText={""}
+        linkText={"Click here to return to the DAOs list page."}
+        onLinkClick={onBackToDAOsLinkClick}
+      />
+    );
+  }
+
+  if (isDAOFound && dao?.isPrivate) {
+    return (
+      <NotFound
+        message={`This DAO is private (${daoAccountId}).`}
+        preLinkText={""}
+        linkText={"Click here to return to the DAOs list page."}
+        onLinkClick={onBackToDAOsLinkClick}
+      />
+    );
+  }
+
+  if (dao && isDAOFound && isSuccess) {
+    const { accountId, type, name } = dao;
+
+    return (
+      <Page
+        gap={0}
+        type={PageLayout.Dashboard}
+        header={
+          <>
+            {notification.isSuccessNotificationVisible && (
+              <Center>
+                <Box padding="16px 80px 0 80px" maxWidth="fit-content" paddingTop="1rem">
+                  <Notification
+                    type={NotficationTypes.SUCCESS}
+                    textStyle="b3"
+                    message={notification.successNotificationMessage}
+                    isLinkShown={true}
+                    linkText="View in HashScan"
+                    linkRef={notification.hashscanTransactionLink}
+                    isCloseButtonShown={true}
+                    handleClickClose={notification.handleCloseNotificationButtonClicked}
+                  />
+                </Box>
+              </Center>
+            )}
+            <DashboardHeader daoAccountId={accountId} name={name} type={type} />
+          </>
+        }
+        body={
+          <Tabs defaultIndex={intialTabIndex} onChange={handleTabChange} isLazy bg={Color.White_02}>
+            <Flex flex="row" padding="0px 80px">
+              <TabList borderBottom="0">
+                {daoNavigationTabs.map((tab, index: number) => {
+                  return (
+                    <Tab
+                      key={index}
+                      tabIndex={index}
+                      height="fit-content"
+                      padding="0"
+                      textStyle="p medium medium"
+                      color={Color.Neutral._400}
+                      stroke={Color.Neutral._400}
+                      _selected={{
+                        color: Color.Neutral._900,
+                        borderBottom: `4px solid ${Color.Primary._500}`,
+                        stroke: Color.Primary._500,
+                      }}
+                    >
+                      <NavLink style={{ padding: "0.75rem 1.25rem" }} to={tab.title.toLowerCase()}>
+                        <Flex alignContent="center" gap={1}>
```

Comment:
The alignment of the tabs and icons needs to precisely match the Figma designs:

Current UI:
<img width="817" alt="Screen Shot 2023-05-28 at 9 52 29 AM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/54907098/ec62ad3a-3a59-48de-a6b3-36c024b789f9">

Figma:
<img width="805" alt="Screen Shot 2023-05-28 at 9 53 58 AM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/54907098/63d51127-bc1f-4929-8d0a-c262de2a4e9b">

Reply:
(Satking should still be **Staking**)

Reply:
Fixed alignment and spacing for the tabs.